gusto/timeloop.py

```diff
@@ -55,14 +55,15 @@ class Timestepper(BaseTimestepper):
     :arg forcing: a :class:`.Forcing` object
     """
 
-    def __init__(self, state, advection_dict, linear_solver, forcing, diffusion_dict=None):
+    def __init__(self, state, advection_dict, linear_solver, forcing, diffusion_dict=None, physics_list=[]):
```

Comment:
IIRC using an empty list as a default argument is dangerous and can lead to weird bugs (@wence- can confirm) - probably best to have default as None then
```
if physics_list is not None:
    self.physics_list = physics_list
else:
    self.physics_list = []
```